diff --git a/node_modules/microbundle/dist/cli.js b/node_modules/microbundle/dist/cli.js
index f3a053a..9139c3f 100755
--- a/node_modules/microbundle/dist/cli.js
+++ b/node_modules/microbundle/dist/cli.js
@@ -317,7 +317,7 @@ var customBabel = (() => {
         }, {
           name: '@babel/plugin-proposal-class-properties',
           loose: true
-        }, !customOptions.modern && !isNodeTarget && {
+        }, false /* we target modern browsers. */ && !customOptions.modern && !isNodeTarget && {
           name: '@babel/plugin-transform-regenerator',
           async: false
         }, {
